src/controllers/noteController.js

```diff
@@ -0,0 +1,107 @@
+const noteModel = require("../models/noteSchema");
+
+const addNote = async (req, res) => {
```

Comment:
Will you write annotations for the route as you did for Authentication?

Reply:
No , No .
I used the mainRouter file .
I think that is the only that i have to used but i documented my code

Reply:
But you created annotations for your PR 1 and 3. Why do you want to do it differently this time?